### PR TITLE
Add 8 aliases to fix dead links

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add 9 aliases to fix dead links

### Which DM version(s) do your changes apply to? (Required)

- [ ] master (the latest development version, including v3.0 changes)
- [ ] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]

File: zh/usage-scenario-simple-migration.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+aliases: ['/docs-cn/tidb-data-migration/stable/usage-scenario-simple-replication/','/docs-cn/tidb-data-migration/v1.0/usage-scenario-simple-replication/','/docs-cn/dev/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.1/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.0/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v2.1/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.0/reference/tools/data-migration/usage-scenarios/simple-replication','/docs-cn/stable/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/zh/tidb-data-migration/stable/usage-scenario-simple-replication/','/zh/tidb-data-migration/stable/usage-scenario-simple-migration']
```
Comment (yikeke) [inline zh/usage-scenario-simple-migration.md:3]:
```suggestion
aliases: ['/docs-cn/tidb-data-migration/stable/usage-scenario-simple-replication/','/docs-cn/tidb-data-migration/v1.0/usage-scenario-simple-replication/','/docs-cn/dev/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.1/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.0/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v2.1/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.0/reference/tools/data-migration/usage-scenarios/simple-replication','/docs-cn/stable/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/zh/tidb-data-migration/stable/usage-scenario-simple-replication/']
```

Comment (yikeke) [reply zh/usage-scenario-simple-migration.md:3]: '/zh/tidb-data-migration/stable/usage-scenario-simple-migration' 应该不需要了

Comment (yikeke) [reply zh/usage-scenario-simple-migration.md:3]: 之前有过 404，后面已经修复了

File: zh/usage-scenario-simple-migration.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+aliases: ['/docs-cn/tidb-data-migration/stable/usage-scenario-simple-replication/','/docs-cn/tidb-data-migration/v1.0/usage-scenario-simple-replication/','/docs-cn/dev/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.1/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.0/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v2.1/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.0/reference/tools/data-migration/usage-scenarios/simple-replication','/docs-cn/stable/reference/tools/data-migration/usage-scenarios/simple-synchronization/']
```
Comment (TomShawn) [inline zh/usage-scenario-simple-migration.md:3]:
```suggestion
aliases: ['/docs-cn/tidb-data-migration/stable/usage-scenario-simple-replication/','/docs-cn/tidb-data-migration/v1.0/usage-scenario-simple-replication/','/docs-cn/dev/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.1/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.0/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v2.1/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/docs-cn/v3.0/reference/tools/data-migration/usage-scenarios/simple-replication','/docs-cn/stable/reference/tools/data-migration/usage-scenarios/simple-synchronization/','/zh/tidb-data-migration/stable/usage-scenario-simple-replication/']
```

**Pull request merged**